references/oidc-mock/apiproxy/proxies/default.xml

```diff
@@ -0,0 +1,242 @@
+        </Flow>
+        <Flow name="GET /certs">
+            <Condition>(proxy.pathsuffix MatchesPath "/certs") and 
+            (request.verb = "GET")</Condition>
+            <Request>
+                <Step>
+                    <Name>AM-SetJWKS</Name>
+                </Step>
+            </Request>
+            <Response/>
+        </Flow>
+        <Flow name="GET /.well-known/openid-configuration">
+            <Condition>(proxy.pathsuffix MatchesPath "/.well-known/openid-configuration") and 
+            (request.verb = "GET")</Condition>
+            <Request>
+                <Step>
+                    <Name>AM-SetDiscoveryDocument</Name>
+                </Step>
+            </Request>
+            <Response/>
+        </Flow>
+        <Flow name="POST /consent">
+            <Condition>(proxy.pathsuffix MatchesPath "/consent") and 
+            (request.verb = "POST")</Condition>
+            <Request>
+                <Step>
+                    <Name>EV-TokenFormParams</Name>
+                </Step>
+                <Step>
+                    <Condition>( oidc.flow.token.state Is null ) or 
+                    ( oidc.flow.token.state = "" )</Condition>
+                    <Name>RF-State-Control</Name>
+                </Step>
+                <Step>
+                    <Condition>( oidc.flow.token.redirect_uri Is null ) or 
+                    ( oidc.flow.token.redirect_uri = "" )</Condition>
+                    <Name>RF-RedirectURI-Control</Name>
+                </Step>
+            </Request>
+            <Response>
+                <Step>
+                    <Name>AM-SetConsentPage</Name>
+                </Step>
+            </Response>
+        </Flow>
+        <Flow name="POST /confirm">
+            <Condition>(proxy.pathsuffix MatchesPath "/confirm") and 
+            (request.verb = "POST")</Condition>
+            <Request>
+                <Step>
+                    <Name>EV-TokenFormParams</Name>
+                </Step>
+                <Step>
+                    <Condition>( oidc.flow.token.state Is null ) or 
+                    ( oidc.flow.token.state = "" )</Condition>
+                    <Name>RF-State-Control</Name>
+                </Step>
+                <Step>
+                    <Condition>( oidc.flow.token.redirect_uri Is null ) or 
+                    ( oidc.flow.token.redirect_uri = "" )</Condition>
+                    <Name>RF-RedirectURI-Control</Name>
+                </Step>
+                <Step>
+                    <Name>JS-GenerateAzCode</Name>
+                </Step>
+            </Request>
+            <Response>
+                <Step>
+                    <Name>AM-SetRedirectLocationToClientApp</Name>
+                </Step>
+            </Response>
+        </Flow>
+        <Flow name="404 - Not Found">
+            <Request>
+                <Step>
+                    <Name>RF-404NotFound</Name>
+                </Step>
+            </Request>
+            <Response/>
```

Comment:
we can delete empty `<Response>` or `<Request>` blocks to make the file easier to read